src/groovy/org/pih/warehouse/core/Constants.groovy

```diff
@@ -107,6 +107,7 @@ class Constants {
     // Default character used to separate columns and newlines in exports
     static final String DEFAULT_COLUMN_SEPARATOR = ","
     static final String CUSTOM_COLUMN_SEPARATOR = ";"
```

Comment:
Perhaps renaming it to `SEMICOLON_COLUMN_SEPARATOR` would be better since now `tab` is also a custom one.